### PR TITLE
Apply linting rules to Cypress tests

### PR DESCRIPTION
**Pull request by jamescrowley**

### CODE REVIEW

File: cypress/integration/productsTests.js [hunk 1]
```diff
@@ -1,67 +1,91 @@
...
+    cy.get("tbody[data-cy=productsTableBody")
+      .find("tr")
+      .its("length")
+      .then(rowsBefore => {
+        cy.get("button[data-cy=deleteProductButton]")
+          .first()
+          .click();
+        cy.get("div[data-cy=deleteConfirmationDialog").should("be.visible");
+        cy.get("button[data-cy=confirmDeleteButton").click();
+        // cy.wait(2000); // updating table takes some time
```
Comment (jamescrowley) [inline cypress/integration/productsTests.js]: @DurkoMatko the cypress linting rules flag this as a 'bad practice' - not sure if there is an alternative way to get it working. I'll take a look once I've figured out the issues on the existing PR with these tests

Comment (DurkoMatko) [reply cypress/integration/productsTests.js]: Yeah it should be done using `.then(()=>{})` which resolves first when the click event finishes all it's "effects"...but it didn't work, the table was still in the old state. I can look at it again but I've actually spend quite some time on it and decided to go just with wait() as it seemed to work fine and I didn't want to loose more time.

File: .travis.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ script:
 - yarn lint
 - cp .travis/.env.feature-tests ./.env.local
 - yarn start &
-- cypress run --record
+- yarn cypress run --record
```
Comment (DurkoMatko) [inline .travis.yml]: :+1:

File: cypress/config.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+export const getTestConfig = () => {
```
Comment (DurkoMatko) [inline cypress/config.js]: :+1:

File: cypress/integration/productsTests.js [hunk 4]
```diff
@@ -60,7 +54,7 @@ describe("CRUD products tests", function() {
             cy.get("div[data-cy=deleteConfirmationDialog").should("be.visible");
             cy.get("button[data-cy=confirmDeleteButton").click();
             cy.wait(2000); // updating table takes some time
-            if (sameNameCountBefore == 1) {
+            if (sameNameCountBefore === 1) {
```
Comment (DurkoMatko) [inline cypress/integration/productsTests.js]: :+1:

**Pull request merged**